image_width: 3840
image_height: 2160
camera_name: default_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [1389.268583, 0.000000, 1837.839971, 0.000000, 1413.839618, 1107.500541, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.233652, 0.033539, 0.000985, 0.004719, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [964.699097, 0.000000, 1790.735049, 0.000000, 0.000000, 1163.240845, 1128.039898, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
transform:
  rows: 1
  cols: 7
  data: [0,0,0,1,0,0,0]
